puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/obal.groovy

```diff
@@ -0,0 +1,22 @@
+def obal(body) {
+
+    def config = [:]
+    body.resolveStrategy = Closure.DELEGATE_FIRST
+    body.delegate = config
+    body()
+
+    def tags = config.tags ? "--tags ${config.tags}" : ""
+    def extra_vars = config.extraVars ?: [:]
+    def extra_vars_file = 'extra_vars.yaml'
+
+    writeYaml file: extra_vars_file, data: extra_vars
+    archive extra_vars_file
+
+    dir('obal') {
+        git url: "https://github.com/theforeman/obal.git", branch: "master"
+    }
+
+    wrap([$class: 'AnsiColorBuildWrapper', colorMapName: "xterm"]) {
+        sh "ANSIBLE_FORCE_COLOR=true PYTHONPATH=obal/ python -m obal.__init__ ${tags} -e @${extra_vars_file} ${config.action} ${config.packages}"
```

Comment:
We should introduce a ```__main__``` so we can use ```python -m obal```

Reply:
https://github.com/theforeman/obal/pull/53

Reply:
tell me more? is it just adding a `__main__.py`?

Reply:
like this?
```python
# cat obal/__main__.py
import obal
if __name__ == '__main__':
    obal.main()
```

Reply:
See my obal PR :)